resources/styles/components/task/details.less

```diff
@@ -0,0 +1,48 @@
+@details-popover-arrow-width: 17px;
+@details-popover-arrow-outer-width: 18px;
+
+.task-details-popover{
+  font-size: 0.8em;
+  max-width: 500px;
+  // THE GREATEST SIN IN CSS!  sorry, just really want the popover height to not matter =(
```

Comment:
this is to override inline styles bootstraps' popover puts on them to position them